site/content/docs/11-deploying.md

```diff
@@ -4,28 +4,25 @@ title: Deployment
 
 Sapper apps run anywhere that supports Node 8 or higher.
 
+### Deploying to Vercel ([formerly ZEIT Now](https://vercel.com/blog/zeit-is-now-vercel))
 
-### Deploying to Now
+#### Using Now v2
 
-> This section relates to Now 1, not Now 2
+We can use an unoffical third-party libary like [now-sapper loader](https://www.npmjs.com/package/now-sapper) to deploy our projects to [Vercel] and even use our sapper runtime environment there.
```

Comment:
@diverent2 the package has been renamed to https://github.com/thgh/vercel-sapper. Mind updating it? (btw, I might remove the text "loader" from the link since it's not part of the repo name)